projects/dotcms-field-elements/src/components/dot-tags/components/dot-autocomplete/dot-autocomplete.tsx

```diff
@@ -1,6 +1,20 @@
 import { Component, Prop, Event, EventEmitter, Element, Watch } from '@stencil/core';
 import autoComplete from '@tarekraafat/autocomplete.js/dist/js/autoComplete';
 
+interface SelectionItem {
+    index: number;
+    value: string;
+    match: string;
+}
+
+interface SelectionFeedback {
```

Comment:
these interfaces should not come from '@tarekraafat/autocomplete.js/dist/js/autoComplete'?

Reply:
No because that's not a ts package, and it doesn't have a types package.